dockers/docker-lldp-sv2/config.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+/bin/cp -rf /etc/swss/lldp/lldpd.j2 /etc/default/lldpd
```

Comment:
probably just rename lldpd.j2 to lldpd

---
In reply to: [96061953](https://github.com/Azure/sonic-buildimage/pull/190#discussion_r96061953) [](ancestors = 96061953)